docs/src/main/sphinx/admin/event-listeners-openlineage.md

```diff
@@ -0,0 +1,133 @@
+
+(openlineage-event-listener-configuration)=
+## Configuration
+
+To configure the OpenLineage event listener plugin, create an event listener properties
+file in `etc` named `openlineage-event-listener.properties` with the following contents
+as an example:
+
+```properties
+event-listener.name=openlineage
+openlineage-event-listener.connect-url=<your openlineage api url>
+openlineage-event-listener.connect-sink=API
+openlineage-event-listener.connect-retry-count=3
+openlineage-event-listener.connect-retry-delay=5s
+openlineage-event-listener.connect-backoff-base=0.5
+openlineage-event-listener.connect-max-delay=1s
+openlineage-event-listener.connect-api-key=__dummy__
+openlineage-event-listener.trino-host=localhost
+openlineage-event-listener.trino-port=8080
+openlineage-event-listener.facets-metadata-enabled=true
+openlineage-event-listener.facets-query-context-enabled=true
+openlineage-event-listener.facets-query-statistics-enabled=true
+openlineage-event-listener.include-query-types=DELETE,INSERT,MERGE,UPDATE,ALTER_TABLE_EXECUTE
+```
+
+And set add `etc/openlineage-event-listener.properties` to `event-listener.config-files`
+in {ref}`config-properties`:
+
+```properties
+event-listener.config-files=etc/openlineage-event-listener.properties,...
+```
+
+### Configuration properties
+
+:::{list-table}
+:widths: 40, 40, 20
+:header-rows: 1
+
+* - Property name
+  - Description
+  - Default
+
+* - openlineage-event-listener.connect-sink
+  - Type of sink to which emit lineage information. Currently only API is supported.
+  - `API`
+
+* - openlineage-event-listener.connect-url
+  - The URI that the plugin will POST events to. Needs to be OpenLineage compatible API with /api/v1/lineage endpoint present.
+  - None. See the [requirements](openlineage-event-listener-requirements) section.
+
+* - openlineage-event-listener.connect-api-key
+  - Api Key used to authenticate within `openlineage-event-listener.connect-url` API.
+  - None.
+
+* - openlineage-event-listener.connect-retry-count
+  - The number of retries on server error. A server is considered to be
+    in an error state when the response code is 500 or higher
+  - `3`
+
+* - openlineage-event-listener.connect-retry-delay
+  - Duration for which to delay between attempts to send a request
+  - `1s`
+
+* - openlineage-event-listener.connect-backoff-base
+  - The base used for exponential backoff when retrying on server error.
+    The formula used to calculate the delay is
+    `attemptDelay = retryDelay * backoffBase^{attemptCount}`.
+    Attempt count starts from 0. Leave this empty or set to 1 to disable
+    exponential backoff and keep constant delays
+  - `2`
+
+* - openlineage-event-listener.connect-max-delay
+  - The upper bound of a delay between 2 retries. This should be
+    used with exponential backoff.
+  - `1m`
+
+* - openlineage-event-listener.trino-host
+  - Trino hostname. Will be used to render Job Namespace.
+  - `localhost`
```

Comment:
I think it should NOT have a default value. And certainly not the generic term `localhost`.